### DISCUSSION TITLE
Apache Superset Row Level Secuirty

### DISCUSSION THREAD

**Discussion by Neo202333** [apache/superset#23580] [General]
Hi all

I'm currently on my journey of learning Apache Superset and now want to embark on using Row level security to limit what users can see on a dashboard. Does anyone have any examples or resource I could use to help setup RLS?

Any help appreciated.

Thanks

**Comment by mdeshmu** [comment]
These docs should help

https://superset.apache.org/docs/security/#row-level-security

https://docs.preset.io/docs/row-level-security-rls
